MariiaHuk/HW5/mh5.2.py

```diff
@@ -0,0 +1,10 @@
+a=0
```

Comment:
good job
but I recommend using more meaningful variable names like as

# Initializing the first two terms of the Fibonacci sequence
current_term = 0
next_term = 1

# Specifying the number of terms to generate
num_terms = 10

# Generating and printing the Fibonacci sequence
for _ in range(num_terms):
    print(current_term)
    
    # Updating variables for the next iteration
    current_term, next_term = next_term, current_term + next_term